client/src/components/page-profile.component.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { AppDispatch, RootState } from '../app/store';
+import { getUserData } from '../features/user/userActions';
+
+export const ProfilePageComponent = () => {
+  const dispatch = useDispatch<AppDispatch>();
+  dispatch(getUserData());
```

Comment:
Very bad practice. Using this out of React life cycle? So then it will executed just on each render?